nilrt_snac/_configs/_config_file.py

```diff
@@ -2,6 +2,9 @@
 
 import pathlib
 import re
+import os
+import grp
```

Comment:
Style: If we haven't already, I know there was some discussion about enforcing code style on check-in. One thing for example is that system imports are sorted alphabetically. I'm not asking you to implement that check, just to be aware of that. See our [general NI python coding standards](https://github.com/ni/python-styleguide/blob/main/docs/Coding-Conventions.md). 

@AlexHearnNI is there a tech debt work item for that enforcement?